js/build.js

```diff
@@ -205,6 +205,16 @@
       }
 
       return new Promise(function(resolve) {
+        var tinymceVersion = tinymce.majorVersion + '.' + tinymce.minorVersion;
+        var plugins = [
+          'advlist', 'lists', 'link', 'image', 'charmap',
+          'searchreplace', 'wordcount', 'insertdatetime', 'table'
+        ];
+
+        if (_.includes(tinymceVersion, '4')) {
```

Comment:
is includes the right approach? what if a future minor/major version had a `4` in it?

Reply:
Yeah, you are right @simonchapman1986, I'll fix this now!